### PR TITLE
Better planning view

### PR DESCRIPTION
**Pull request by hatch01**
The planning view is now a day view and there is a new week view. 
month view soon ;)

### CODE REVIEW

File: apps/onyx/lib/core/extensions/date_extension.dart [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      case 9:
+        return short ? "Sept." : "Septembre";
+      case 10:
+        return short ? "Oct." : "Octobre";
+      case 11:
+        return short ? "Nov." : "Novembre";
+      case 12:
+        return short ? "Déc." : "Décembre";
+      default:
+        return "Erreur";
```
Comment (Persson-dev) [inline apps/onyx/lib/core/extensions/date_extension.dart]: I think you could do the same here as for toWeekDayName : declaring constants and then using them

File: apps/onyx/lib/core/widgets/double_scrollable_widget.dart [hunk 2]
```diff
@@ -0,0 +1,151 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+/// don't forget to add NeverScrollableScrollPhysics() to all the scrollable widgets
```
Comment (Persson-dev) [inline apps/onyx/lib/core/widgets/double_scrollable_widget.dart:4]: Did you forget or is it good ?

**Comment by Persson-dev** [comment]
Also, the html does not render in emails on this branch

**Comment by hatch01** [comment]
> Also, the html does not render in emails on this branch

Yes but it should work after merging. Because I do shit with branch

File: apps/onyx/lib/core/widgets/double_scrollable_widget.dart (hunk 2 above)
Comment (hatch01) [reply apps/onyx/lib/core/widgets/double_scrollable_widget.dart:4]: It's is a note for using this widget. So it is good.

File: apps/onyx/lib/core/extensions/date_extension.dart (hunk 1 above)
Comment (hatch01) [reply apps/onyx/lib/core/extensions/date_extension.dart]: done

**Review by Persson-dev — APPROVED** [review]

**Pull request merged**